index.js

```diff
@@ -274,20 +316,31 @@ module.exports = function (log, indexesPath) {
           saveCoreIndex('sequence', indexes['sequence'], count)
 
         index.seq = indexes['offset'].seq
-        if (indexNeedsUpdate) saveIndex(op.data.indexName, index)
+        if (indexNeedsUpdate) {
+          if (index.prefix) savePrefixIndex(op.data.indexName, index, count)
+          else saveIndex(op.data.indexName, index)
+        }
 
         cb()
       },
     })
   }
 
-  function createIndexes(missingIndexes, cb) {
+  function createIndexes(orphanOps, cb) {
```

Comment:
I actually like the missingIndexes name better. I'm not exactly sure what orphan means here.

Reply:
Oh, let me explain (and I'm open to changing it back): I general take the name to indicate the type of the thing, and in this case `missingIndexes` was not an array of indexes, it was an array of `op.data`. This confused me for a while, and changing the name helped me to know that these were actually ops, and they were passed to other functions like `updateIndexValue(opData, .....)`

Reply:
How about opsNotIndexed instead then?

Reply:
Even better